cmd/gaiad/cmd/testnet_set_local_validator.go

```diff
@@ -266,6 +266,9 @@ func updateApplicationState(app *gaia.GaiaApp, args valArgs) error {
 
 	app.SlashingKeeper.SetValidatorSigningInfo(appCtx, newConsAddr, newValidatorSigningInfo)
 
+	// PROVIDER
+	app.ProviderKeeper.DeleteLastProviderConsensusValSet(appCtx)
+
 	shortVotingPeriod := time.Second * 20
```

Comment:
The section below refers to the Gov module, right?

```suggestion
        // GOVERNANCE
	shortVotingPeriod := time.Second * 20
```